libraries/botbuilder-core/src/showTypingMiddleware.ts

```diff
@@ -98,7 +103,7 @@ export class ShowTypingMiddleware implements Middleware {
 
         // Let the rest of the process run.
         // After everything has run, stop the indicator!
-        return await next().then(stopInterval, stopInterval);
+        return await next().then(stopInterval, stopIntervalWithError);
```

Comment:
discussed with @Stevenic and his suggested fix was more like this:

```ts
        try
        {
              return await next();
         }
        finally
        {
               stopInterval();
        }
```

i.e. similar to the C#

Reply:
@johnataylor  Wouldn't a try/finally still swallow all exceptions that occur within 'next()'?  The goal of the fix is to propagate those exceptions up the stack, not catch and swallow them.

Reply:
@ERicDahlvang that's not my understanding. this should behave exactly like the finally in C#. in other words adding a catch will stop the exception but a finally just gets run as the exception is propagated

Reply:
Oh, there's no catch.  Sorry, I missed that... carry on. :)